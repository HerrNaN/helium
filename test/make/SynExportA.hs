module SynExportA where

import SynExportB

type P = Int

main = f 3
