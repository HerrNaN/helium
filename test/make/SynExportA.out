Prelude is up to date
Compiling make/SynExportC.hs
Compilation successful
Compiling make/SynExportB.hs
(8,1): Warning: Missing type signature: x :: A
Compilation successful with 1 warning
Compiling make/SynExportA.hs
(7,1): Warning: Missing type signature: main :: Int
Compilation successful with 1 warning
4
